src/main/java/com/kajan/iworkflows/controller/FileController.java

```diff
@@ -28,28 +31,48 @@
 
     private NextcloudService nextcloudService;
     private DummyUserStoreServiceImpl userStoreService;
+    private LearnOrgServiceImpl learnOrgService;
+
+    @Value("${testing}")
+    private Boolean testing;
```

Comment:
@RamiyaRatnarajah  did you check the file upload in test mode and with learnorg service?

Reply:
ya checked. Its uploaded in nextcloud. But error is coming in handleAttachment class.

Reply:
Can you post the error?